backend/src/v2/driver/driver.go

```diff
@@ -217,6 +244,27 @@ func Container(ctx context.Context, opts Options, mlmd *metadata.Client, cacheCl
 	if err != nil {
 		return nil, err
 	}
+
+	// When the container image is a dummy image, there is no launcher for this task.
+	// This happens when this task is created to implements a Kubernetes-specific configuration, i.e., there is nothing to run.
+
+	// This task creates a PVC.
+	var pvcName string
+	if opts.Container.Image == "argostub/createpvc" {
```

Comment:
There are a number of places we use the string value `"argostub/createpvc"` directly. We should create a constant for it.

Reply:
I will create a constant in the `driver` package for now, but I think in the long run it would be nice to declare them in the proto, and import from there.